source/includes/examples/_find-the-center.html.md

```diff
@@ -0,0 +1,182 @@
+# Find the Center Example
+
+In this example, we'll walk you through the various statements that are part of the Find the Center project, including the simulator file and the Inkling file. Find the Middle was created as a very basic example of Inkling and how to connect to a simulator.
+
+The general premise of Find the Center is that the computer (via Inkling) takes a step (move) of -1, 0, or 1, and gets rewarded for staying in the center (1) for each round. The max is 2 and the min is 0.
+
+## Inkling File
+
+###### Schema
+
+```inkling
+schema GameState
+    Int32 value
+end
+```
+
+The `GameState` schema has one record, `value`, assigned the constrained type of Int32.
+
+```inkling
+schema PlayerMove
+    Int32{-1, 0, 1} delta
+end
+```
+
+The `PlayerMove` schema has one record, `delta`, assigned the constrained type of Int32 to allow for negative and positive values.
+
+```inkling
+schema SimConfig
+    Int32 dummy
+end
+```
+
+The `SimConfig` schema has one record, `dummy`, because there is no configuration needed in this particular example.
+
+###### Concept
+
+```inkling
+concept find_the_center
+    is classifier
+    predicts (PlayerMove)
+    follows input(GameState)
+    feeds output
+end
+```
+
+This concept is named `find_the_center`, and it takes input from the simulator about the state of the game (`GameState` schema). It outputs to the `PlayerMove` schema. This is the AI's next move in the simulation.
+
+###### Simulator
+
+```inkling
+simulator find_the_center_sim(SimConfig)
+    action (PlayerMove)
+    state (GameState)
+end
+```
+
+The `simulator` is called `find_the_center_sim` (shown in #simulator-file) and takes the schema input of `SimConfig` (even though it isn't configuring anything, it's required by the simulator). The simulator is going to output the `action` and the `state` of the simulator which is sent to the lesson.
```

Comment:
The simulator is going to output the `action` and the `state` of the simulator which is sent to the lesson.

should be:
The `find_the_center` concept will be trained using the `find_the_center_sim` simulator. To define the training relationship between the simulator and the concept we must begin by defining the simulator. `find_the_center_sim` expects an `action` defined in the `PlayerMove` schema as input and replies with a `state` defined in the `GameState` schema as output.